### PR TITLE
Issue #34, option to hide status bar in general settings

### LINKED ISSUE #34: Translucent status bar

### PR DESCRIPTION
**Pull request by rosenpin**
Added option in general settings to hide status bar.

fixes #34

### CODE REVIEW

**Comment by okungnyo** [comment]
Awesome work!

File: app/src/main/java/me/ccrama/redditslide/Activities/SuperBaseActivity.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        translucent();
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        translucent();
+    }
+
+    public void translucent() {
```
Comment (l3d00m) [inline app/src/main/java/me/ccrama/redditslide/Activities/SuperBaseActivity.java]: You should probably do it only if Reddit.translucentStatusBar is true.

**Comment by l3d00m** [comment]
We should also extend the BaseActivity in the future so that it's a real base activity.

File: app/src/main/java/me/ccrama/redditslide/Activities/SuperBaseActivity.java (hunk 1 above)
Comment (rosenpin) [inline app/src/main/java/me/ccrama/redditslide/Activities/SuperBaseActivity.java]: You're absolutely right.

**Comment by ccrama** [comment]
What's the difference between BaseActivity and SuperBaseActivity?

**Comment by rosenpin** [comment]
Base activity isn't really a base activity, or adds swipe back and some more things,  superbaseactivity extends app compact activity and doesn't add anything but app wide features.

**Comment by recalculated** [comment]
I think it would be better to revise the BaseActivity so that all activities can extend it, and then implement the feature in BaseActivity. It doesn't really make sense to have a BaseActivity that isn't really a base, and then create another SuperBaseActivity that does something that should just go in the actual base activity.

**Comment by l3d00m** [comment]
@rosenpin BaseActivity is now a real baseactivity, would you like to add this again?

**Comment by The0x539** [comment]
wait, why

**Pull request closed without merging**